conbench/entities/benchmark_result.py

```diff
@@ -391,8 +403,27 @@ class _BenchmarkFacadeSchemaCreate(marshmallow.Schema):
         },
     )
     batch_id = marshmallow.fields.String(required=True)
-    timestamp = marshmallow.fields.DateTime(
-        required=True, metadata={"description": "Timestamp the benchmark ran"}
+
+    # `AwareDateTime` with `default_timezone` set to UTC: naive datetimes are
+    # set this timezone.
+    timestamp = marshmallow.fields.AwareDateTime(
+        required=True,
+        format="iso",
+        default_timezone=timezone.utc,
+        metadata={
+            "description": conbench.util.dedent_rejoin(
+                """
+                A datetime string indicating the time at which the benchmark
+                was started. Expected to be in ISO 8601 notation.
+                Timezone-aware notation recommended. Timezone-naive strings are
+                interpreted in UTC. Fractions of seconds can be provided but
+                are not returned by the API. Example value:
+                2022-11-25T22:02:42Z. This timestamp defines the default
+                sorting order when viewing runs via the UI or when enumerating
+                runs via the /api/runs/ HTTP endpoint.
```

Comment:
I think `Run.timestamp` is what determines the sort order of the runs UI (aka the index) and `/api/runs`, not `BenchmarkResult.timestamp`. 

`BenchmarkResult.timestamp` only determines the sort order of `/api/benchmarks` with no argument, and the `/benchmarks` webpage (which I suspect not many people visit).

Reply:
Yeah, I'll second what Austin said here: the sort order here would (likely) be on any list of benchmarks (like when displaying one run in the UI). Maybe we could say something along those lines "timestamps are used as the default order when listing items that contain them (benchmarks, runs, etc.) in the API and the UI" which helps explain without being too specific about which endpoints (so if we add some we don't need to come back and update this without it still being correct).

Reply:
> Run.timestamp is what determines the sort order of the runs UI (aka the index) and /api/runs, not BenchmarkResult.timestamp

Thanks! I confused those once again when writing the text.